ext/-test-/string/chilled.c

```diff
@@ -0,0 +1,13 @@
+#include "ruby.h"
+
+static VALUE
+bug_s_rb_str_chilled_p(VALUE self, VALUE str)
+{
+    return INT2FIX(rb_str_chilled_p(str));
```

Comment:
```suggestion
    return rb_str_chilled_p(str) ? Qtrue : Qfalse;
```
seems nicer because then it returns a boolean to Ruby and not an Integer.

Reply:
Why thought? It's meant meant to be used from C. I don't see the value of returning `Qtrue / Qfalse` over `1/0` as it requires an extra `RTEST()`.

Reply:
I was commenting about the test, not the impl of rb_str_chilled_p, but looks like you saw it as well after.